ui/app/AppLayouts/Profile/stores/WalletStore.qml

```diff
@@ -21,9 +20,7 @@ QtObject {
     property var importedAccounts: walletSectionAccounts.imported
     property var generatedAccounts: walletSectionAccounts.generated
     property var watchOnlyAccounts: walletSectionAccounts.watchOnly
-
-    property var currencyStore: SharedStore.RootStore.currencyStore
```

Comment:
was this causing a warning?

Reply:
Yes, I've been having 3 Cyclic depenencies and this is one of them:
```
Warning: Cyclic dependency detected between "qrc:/app/AppLayouts/Wallet/stores/RootStore.qml" and "qrc:/app/AppLayouts/Profile/stores/WalletStore.qml" (:0, )
Warning: Cyclic dependency detected between "qrc:/imports/shared/stores/RootStore.qml" and "qrc:/app/AppLayouts/Profile/stores/WalletStore.qml" (:0, )
Warning: Cyclic dependency detected between "qrc:/app/AppLayouts/Wallet/stores/RootStore.qml" and "qrc:/app/AppLayouts/Wallet/panels/SelectGeneratedAccount.qml" (:0, )
```

Reply:
ugh